Source/Scene/Label.js

```diff
@@ -44,18 +47,30 @@ import VerticalOrigin from './VerticalOrigin.js';
     }
 
     function parseFont(label) {
-        var div = document.createElement('div');
-        div.style.position = 'absolute';
-        div.style.opacity = 0;
-        div.style.font = label._font;
-        document.body.appendChild(div);
-
-        label._fontFamily = getCSSValue(div,'font-family');
-        label._fontSize = getCSSValue(div,'font-size').replace('px', '');
-        label._fontStyle = getCSSValue(div,'font-style');
-        label._fontWeight = getCSSValue(div,'font-weight');
-
-        document.body.removeChild(div);
+        var fontInfo = fontInfoCache[label._font];
+        if (!defined(fontInfo)) {
+            var div = document.createElement('div');
+            div.style.position = 'absolute';
+            div.style.opacity = 0;
+            div.style.font = label._font;
+            document.body.appendChild(div);
+
+            fontInfo = {
+                family : getCSSValue(div, 'font-family'),
+                size : getCSSValue(div, 'font-size').replace('px', ''),
+                style : getCSSValue(div, 'font-style'),
+                weight : getCSSValue(div, 'font-weight')
+            };
+
+            document.body.removeChild(div);
+            if (Object.keys(fontInfoCache).length < fontInfoCacheMaxSize) {
```

Comment:
Not a huge deal, but to reduce memory allocations it might be better to track the cache size in a separate variable rather than using `Object.keys`.